README.md

```diff
@@ -60,14 +67,31 @@ This repo contains some tooling to help provision the factory-fresh devices into
 
 ### Build and release
 
+Each of the four firmware components listed above are built and released by a staticly-configured pipeline, and ultimately make their way into a public Firmware Transparency log.
+
 The [deployment/build_and_release](deployment/build_and_release) directory contains Terraform configs to define Cloud Build triggers which build and release the firmware and recovery image.
 
 TODO(jayhou): add public links.
 
-Every firmware version and recovery image version is written to the transparency log. The software itself is written to the artefacts log. The filename is the SHA256 of the software.
+### Transparency
 
-### Claimant Model
+Given how important the role witnessing is to the security properties of transparency-enabled ecosystem, it's important that the operation of the witnesses, and therefore the software running on the devices, is as open to inspection and verification as possible.
 
+We have embodied this principle into the design of the ArmoredWitness firmware and tooling:
+
+* All firmware is opensource, written in TamaGo, and is build-reproducible from source.
+* All firmware is logged to a Firmware Transparency log at build and release time.
+* The [`provision`](cmd/provision/) tool will only use firmware artefacts discovered in the FT log in order to program devices.
+* The on-device self-update process requires that updated firmware is hosted in the FT log.
+* The boot "chain of trust" requires valid "off-line FT proof bundles" to be present alongside the firmware at boot time:
```

Comment:
Nit, the sub-bullets are inconsistent in:
 - case of the starting entries.
 - Whether they end in a period